projects/centos-7/Dockerfile

```diff
@@ -1,29 +1,31 @@
 FROM centos:7
 
 RUN yum update -y && \
-    yum install -y \
-        which \
-        epel-release \
-        wget \
-        python \
-        python-psutil \
-        python-requests \
-        python-jsonschema \
-        python-pip \
-        rsyslog \
-        gnupg \
-        unixODBC-devel \
-        authd
+  yum install -y \
```

Comment:
why the realignment?

Reply:
The ubuntu Dockerfiles are aligned this way. Although, I see now that they are not completely equivalent.

Reply:
yeah, leave the indentation/whitespace equal, i think it's easier to read.

Reply:
Meaning, you'd prefer...
```Dockerfile
# 1
RUN yum update -y && \
    yum install -y \
        which \
        ...
```
to
```Dockerfile
# 2
RUN \
  yum update -y && \
  yum install -y \
    which \
    ...
```

Reply:
i think i like 1 better.    the rest of the file seems to do that...

Reply:
Right, but in ubuntu's Dockerfiles they are not formatted this way. If we only care about consistency within the files, that's fine, but I just wanted to point out that there is a difference between the platforms on this front.

Reply:
Actually, the centos:7 Dockerfiles in the development environment are also formatted like the ubuntu files (2): https://github.com/irods/irods_development_environment/blob/main/Dockerfile.irods_core_builder.centos7

Reply:
what a mess.    yeah, pick one and run with it.